code/modules/antagonists/slaughter/slaughter.dm

```diff
@@ -71,7 +71,11 @@
 /mob/living/simple_animal/slaughter/phasein()
 	. = ..()
 	speed = 0
-	boost = world.time + 60
+	boost = world.time + 20
```

Comment:
Mark your changes ![like this](https://github.com/yogstation13/tg-rebase/blob/master/yogstation/README.md)